Assets/Scripts/UI/MainMenu.cs

```diff
@@ -12,7 +12,7 @@ private void Start()
         if (versionsText != null && GameControl.control != null)
         {
             // Set the text of the text field to the game version from GameControl
-            versionsText.text = "Version: " + GameControl.control.gameVersion;
+            versionsText.text = "Version: " + GameControl.control.profileVersion + GameControl.control.gameVersion;
```

Comment:
Change the string to a literal (stringbuild) line:

versionsText.text = $"Version: {GameControl.control.profileVersion}{GameControl.control.gameVersion}";

Reply:
Done, thanks for the info